src/index.js

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './Calculator';
+// import App from './App';
+// import reportWebVitals from './reportWebVitals';
```

Comment:
Please be sure to get rid of all the comments from your code 📝 This helps by making your PRs more professional and to meet the requirements 🧐